README.md

```diff
@@ -2,33 +2,66 @@
 
 # Stylist
 
-The immediate need was for a code style checking tool which understood Fortran.
-Such things being quite thin on the ground. However very few large software
-projects are implemented in a single language. Thus a tool was developed which
-provided a framework to aid in delivering support for multiple languages and
-multiple style choices.
+Stylist is a code style checking tool built on a framework which supports
+multiple styles across multiple languages.
 
-The framework supports multiple styles, each consisting of multiple rules.
-Each of these may be one of two types, those which treat the source as a text
-file and those which treat it as a parse tree.
+But aren't there many code style checking tools out there, why create a new
+one?
+
+The simple reason is that few of them support Fortran, a language still in
+widespread use in the scientific computing domain. They can also tend towards
+the zealous when enforcing the "one true style" whereas long running science
+models tend to diverge quite radically in what they like, style-wise.
+
+Large models also tend to be implemented using more than one language so a
+tool which is capable of understanding all of them would be welcomed by
+science developers.
+
+Thus a tool was developed based on a framework which hopes to support ready
+extension and reconfiguration.
 
 The project is still in its infancy so only a few rules have been implemented
-and only for fortran.
+and only for Fortran. Stubs are provided to show how it might be extended to
+support C.
+
+Find the project at https://github.com/MetOffice/stylist
+
+## Installation
+
+Installation can be as simple as `pip install stylist`.
```

Comment:
```suggestion
Installation can be as simple as `pip install stylist` or `conda install -c conda-forge stylist`.
```

Reply:
Happy to oblige. I'd forgotten about Conda.